src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.Designer.cs

```diff
@@ -736,7 +736,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to The package is deprecated. Use {0} instead..
+        ///   Looks up a localized string similar to This package version is deprecated. Use {0} instead..
```

Comment:
Change isn't related, but the designer automatically regenerated this string which somehow was missed in a previous change to the file.